### ISSUE TITLE
[enhancement] memory management from STUDY

### ISSUE THREAD

**Issue by CPernet** [sccn/bids-matlab-tools#113]
If one have a massive dataset, because one relies on STUDY it becomes a problem as STUDY loads every .set (as a work around of course we can do several small STUDY(ies) and re-assemble) but it would be nice have a more direct solution

**Comment by CPernet** [comment]
also for ongoing project, it would be nice to just be able to update a STUDY and it would just create new fles (write now redo it all)

**Comment by arnodelorme** [comment]
Good point. I will look into this if the datasets are in EEGLAB format (in that case, there is no need to load the data).
